backend/src/test/java/com/handong/rebon/acceptance/tag/TagReadAcceptanceTest.java

```diff
@@ -0,0 +1,59 @@
+package com.handong.rebon.acceptance.tag;
+
+import com.handong.rebon.acceptance.AcceptanceTest;
+import com.handong.rebon.acceptance.admin.AdminTagRegister;
+import com.handong.rebon.tag.domain.Tag;
+import io.restassured.RestAssured;
+import io.restassured.common.mapper.TypeRef;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TagReadAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    private AdminTagRegister adminTagRegister;
+    private Map<String, Tag> tags = new HashMap<>();
+
+    @BeforeEach
+    void setUp() {
+        tags = adminTagRegister.register("북구", "남구");
+    }
+
+    @Test
+    @DisplayName("모든 태그 조회하기")
+    public void getAllTags() {
+        //given when
+        ExtractableResponse<Response> response = 태그_조회_요청();
+        List<Tag> result = response.as(new TypeRef<>() {
+        });
```

Comment:
api 요청의 결과값이 TagResponse여서 List<TagResponse>로 받아주는게 좋아보입니다.

Reply:
아 혹시 지금도 list로 받아지고 있는게 아닌가요?!ㅠ 정확히 어떻게 바꾸어야 할지 모르겠습니다ㅠ!

Reply:
인수테스트는 요청부터 반환까지를 테스트하는 것입니다.
`GET /api/tags`라는 요청을 보냈을 때 `List<TagResponse>` 타입을 받는 걸 원하는데 현재는 그냥 도메인으로 받아오고 있습니다.(`List<Tag>`)

```suggestion
        List<TagResponse> result = response.as(new TypeRef<>() {
        });
```